src/schema/workflow_execution_activity.yaml

```diff
@@ -52,6 +52,7 @@ classes:
         required: true
       was informed by:
         required: true
+        range: omics processing
```

Comment:
Are you wanting the range to be just omics processing or the more general biosample processing? Also should we make biosample process and kind of prov:Activity?

Reply:
@cmungall The PR fails b/c `nmdc` is not imported, and `omics processing` is a class in `nmdc`. Here are the imports for `workflow_execution_activity`:
```
imports:
  - prov
  - core
  - external_identifiers
  ```
However, `nmdc` also imports `workflow_execution_activity`. Here are its imports:
```
imports:
  - linkml:types
  - mixs
  - core
  - prov
  - workflow_execution_activity
  - annotation
  - external_identifiers
```

So, if `workflow_execution_activity` imports `nmdc` and `nmdc` imports `workflow_execution_activity` won't this create a cyclic dependency?